contracts/staking/stakeManager/StakeManager.sol

```diff
@@ -356,7 +357,8 @@ contract StakeManager is IStakeManager, StakeManagerStorage {
         totalStaked = totalStaked.sub(amount);
 
         NFTContract.burn(validatorId);
-        signerToValidator[validators[validatorId].signer] = INCORRECT_VALIDATOR_ID;
+        signerToValidator[validators[validatorId]
+            .signer] = INCORRECT_VALIDATOR_ID;
```

Comment:
Can you configure the linter so it doesn't break this line? It is probably max width parameter or something.

Reply:
what should be the max length?